PowerEditor/installer/nativeLang/swedish.xml

```diff
@@ -1,1263 +1,1491 @@
+            <DoReloadOrNotAndLooseChange title="Ladda om" message="&quot;$STR_REPLACE$&quot;
 
-Denna fil har modifierats av ett annat program.
-Vill du ladda om den och förlora ändringarna i Notepad++?"/>
-			<PrehistoricSystemDetected title="Förhistoriskt system noterades" message="Det verkar som om du fortfarande använder ett förhistoriskt system. Vi är ledsn, men den här funktionen fungerar endast på ett modernt system."/>
-			<XpUpdaterProblem title="Uppdaterare för Notepad++" message="Uppdateraren för Notepad++ är inte kompatibel med XP på grund av ett föråldrat säkerhetslager i XP.
-Vill du gå till hemsidan för Notepad++ för att ladda ned den senaste versionen?"/>
-			<DocTooDirtyToMonitor title="Övervakningsproblem" message="Dokumentet är ändrat. Spara ändringarna innan du övervakar den."/>
-			<DocNoExistToMonitor title="Övervakningsproblem" message="Filen måste finnas för att kunna övervakas."/>
-			<FileTooBigToOpen title="Problem med filstorlek" message="Filen är för stor för att öppnas med Notepad++"/>
-			<CreateNewFileOrNot title="Skapa ny fil" message="&quot;$STR_REPLACE$&quot; existerar ej. Skapa den?"/>
-			<CreateNewFileError title="Skapa ny fil" message="Kan inte skapa filen &quot;$STR_REPLACE$&quot;."/>
-			<OpenFileError title="FEL" message="Kan inte öppna filen &quot;$STR_REPLACE$&quot;."/>
-			<FileBackupFailed title="Misslyckad filbackup" message="Föregående version av filen kunde inte spara i backup-mappen &quot;$STR_REPLACE$&quot;.
+Denna fil har ändrats av ett annat program.
+Vill du ladda om den och förlora ändringarna som gjordes i Notepad++?"/>
+            <PrehistoricSystemDetected title="Förhistoriskt system upptäcktes" message="Det verkar som om du fortfarande använder ett förhistoriskt system. Tyvärr fungerar endast denna funktion på ett modernt system."/> <!-- HowToReproduce: Launch "Document Map" under Windows XP. -->
+            <XpUpdaterProblem title="Uppdaterare för Notepad++" message="Uppdateraren för Notepad++ är inte kompatibel med XP på grund av ett föråldrat säkerhetslager i XP.
+Vill du gå till hemsidan för Notepad++ för att ladda ned den senaste versionen?"/>  <!-- HowToReproduce: Run menu "? -> Update Notepad++" under Windows XP. -->
+            <GUpProxyConfNeedAdminMode title="Proxyinställningar" message="Var god starta om Notepad++ som administratör för att konfigurera proxyn."/>
+            <DocTooDirtyToMonitor title="Övervakningsproblem" message="Dokumentet har ändrats. Spara ändringarna innan du övervakar den."/>
+            <DocNoExistToMonitor title="Övervakningsproblem" message="Filen måste finnas för att kunna övervakas."/>
+            <FileTooBigToOpen title="Problem med filstorleken" message="Filen är för stor för att öppnas med Notepad++"/> <!-- HowToReproduce: Try to open a 4GB file (it's not easy to reproduce, it depends on your system). -->
+            <CreateNewFileOrNot title="Skapa ny fil" message="&quot;$STR_REPLACE$&quot finns inte. Vill du skapa den?"/>
```

Comment:
Build failed due to xml error. Missing semicolon at second `&quot;`.

Reply:
Done